packages/mui-base/src/AutocompleteUnstyled/useAutocomplete.d.ts

```diff
@@ -292,6 +292,13 @@ export interface UseAutocompleteProps<
   value?: AutocompleteValue<T, Multiple, DisableClearable, FreeSolo>;
 }
 
+export interface UseAutocompleteParameters<
```

Comment:
Aliasing the interface so that it matches the name pattern used for the hooks parameters.